lib/honeybadger/splitproject.rb

```diff
@@ -1,10 +1,22 @@
 require 'honeybadger/honeybadger'
 require 'honeybadger/key_helpers'
+require 'honeybadger/team_filter'
 
 module Honeybadger
   module Splitproject
-    
+    @@honeybadger_filters = [TeamFilter.new]
+
     class << self
+
+      def filters
+        @@honeybadger_filters
+      end
+
+      def add_filter(klass)
+        raise "MissingFilterMethod" unless klass.respond_to?(:filter)
```

Comment:
Out of curiosity, what other filters were you thinking could be added anyway? Or are you just leaving it open for now?